src/redux/createState.js

```diff
@@ -0,0 +1,12 @@
+import { applyMiddleware, combineReducers, createStore } from 'redux';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import booksReducer from './books/books';
+
+const reducer = combineReducers({
+  booksReducer,
+});
+
+const store = createStore(reducer, applyMiddleware(logger, thunk));
+
+export default store;
```

Comment:
- [ ] As the previous reviewer said that you should be following the project requirements, The structure of the files should follow the ducks pattern:

```
/src
        |---/components
        |---/redux
            |--- /books
                    | books.js
            |--- /categories
                    | categories.js
            | cofigureStore.js
        | index.js
```
- [ ] Also the file should be named configureStore.js.
- [ ] Also it should Imports `categories` from `src/redux/categories`.
     * It will be something like this: `import categories from './categories/categories';`.